client/allocrunner/taskrunner/logmon_hook.go

```diff
@@ -99,6 +103,11 @@ func reattachConfigFromHookData(data map[string]string) (*plugin.ReattachConfig,
 func (h *logmonHook) Prestart(ctx context.Context,
 	req *interfaces.TaskPrestartRequest, resp *interfaces.TaskPrestartResponse) error {
 
+	if h.isLoggingDisabled() {
+		h.logger.Info("logging is disabled by driver")
```

Comment:
might wanna drop this to debug level? - It's likely to be noisy and very repetitive?